client/components/rewind-credentials-form/index.jsx

```diff
@@ -79,11 +79,22 @@ export class RewindCredentialsForm extends Component {
 			...this.state.form,
 		};
 
+		let userError = '';
+
+		if ( ! payload.user ) {
+			userError = translate( 'Please enter your server username.' );
+		} else if ( 'root' === payload.user ) {
+			userError = translate(
+				"We can't accept credentials for the root user. " +
```

Comment:
we _can_ but we _won't_

might need to update the messaging so as not to introduce the question, "why can't you?"

not a big deal

Reply:
"won't" seems even more likely to beg that question to me...